lib/index.js

```diff
@@ -192,6 +193,14 @@ api.set = async ({
   }
   if(secret) {
     rValue.secret = secret;
+    // produces a URL like `otpauth://totp/...`
+    rValue.otpAuthUrl = authenticator.keyuri(
+      // user identifier
+      email || account,
```

Comment:
Do you think we should always bind this to `account` instead? (which means looking it up before hand if we don't have the ID).

Reply:
This is added here for the benefit of the authenticator APP that uses the URL, I don't know how it's used.  However, we can assume that the authenticator App may have knowledge of email addresses, but certainly would not know anything about the account ID (urn:uuid:...)